.ci/install_qemu.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+QEMU_REPO="github.com/qemu/qemu"
+QEMU_DIR="qemu"
+KATA_QEMU_BRANCH="stable-2.11"
+PACKAGING_REPO="github.com/kata-containers/packaging"
+QEMU_CONFIG_SCRIPT="configure-hypervisor.sh"
+
+git clone "https://${QEMU_REPO}"
+
+# Get qemu configuration script and copy to
+# the qemu repository
+go get -d "$PACKAGING_REPO" || true
+cp "${GOPATH}/src/${PACKAGING_REPO}/scripts/${QEMU_CONFIG_SCRIPT}" "${QEMU_DIR}"
+
+pushd "$QEMU_DIR"
+git checkout "$KATA_QEMU_BRANCH"
+
+echo "Build Qemu"
+# Patch to not fail on --disable-static
+curl https://raw.githubusercontent.com/clearcontainers/packaging/master/qemu-lite/configure.patch | patch -p1
```

Comment:
unknown/unsupported options should not be ignored, using https://github.com/kata-containers/qemu/blob/stable-2.11/configure.sh this patch is not needed

Reply:
@devimc we have agreed to use github.com/qemu/qemu instead of the kata-containers/qemu repo and use https://github.com/kata-containers/packaging/blob/master/scripts/configure-hypervisor.sh as the configurion options. 
I have opened https://github.com/kata-containers/packaging/pull/12 to fix this, then I can remove the patch, here.